build.sbt

```
@@ -33,11 +30,12 @@ resolvers ++= Seq(
   Resolver.bintrayRepo("webjars", "maven"),
   Resolver.typesafeRepo("releases"),
   Resolver.sbtPluginRepo("releases"),
-  Resolver.sonatypeRepo("releases"),
   Resolver.mavenLocal
 )
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-web" % "1.4.4")
+resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+
+addSbtPlugin("com.github.sbt" % "sbt-js-engine" % "1.3.0")
```

```suggestion
addSbtPlugin("com.github.sbt" % "sbt-js-engine" % "1.3.5-M1")
```

@platypii Please give this a try (as soon as it is available at https://repo1.maven.org/maven2/com/github/sbt/sbt-js-engine_2.12_1.0/) and let me know if it fixes your problem, see https://github.com/sbt/sbt-js-engine/pull/66#issuecomment-1823581550
Thanks!

---

Confirmed that 1.3.5-M1 does fix the performance regression, and now things are super fast! THANK YOU!! 👏

---

Great! 1.3.5 available now :wink: